src/thunderbots/software/test/test_util/util.cpp

```diff
@@ -0,0 +1,83 @@
+#include "util.h"
+
+namespace UnitTest
+{
```

Comment:
We should try and make sure namespaces match the filepath, makes things a lot easier to think about (unless we've used the `UnitTest` namespaces elsewhere and are trying to be consistent?).

Reply:
Agree. Fixed now to `Test::TestUtil`

Reply:
I would honestly name them exactly the same, but this seems to conflict with our naming conventions (ie. CamelCase for namespaces and snake_case for filenames). I should've noticed this a _long_ time ago, but I feel filenames should match the class/namespace. What do you think? Is it still reasonable to change this?